src/MVTFeature.js

```diff
@@ -87,7 +87,7 @@ function ajaxCallback(self, response) {
   }
 
   self._setStyle(self.mvtLayer.style);
-  redrawTiles(self);
+  this.redraw();
```

Comment:
in this context, `this.redraw` is undefined; it should use `self.redraw` instead